release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,291 @@
+
+[options="header"]
+|====================================================================================================================
+| Counter     | Destination                                                           
+| Clusters    | Platform Configuration → Clusters                                      
+| Nodes       | Configuration Management → Application & Infrastructure → Nodes        
+| Violations  | Violations main menu                                                   
+| Deployments | Configuration Management → Application & Infrastructure → Deployments  
+| Images      | Vulnerability Management → Dashboard → Images                         
+| Secrets     | Configuration Management → Application & Infrastructure → Secrets      
+|====================================================================================================================
+
+[id="dashboard-filter"]
+==== Dashboard filter
+
+The dashboard now includes a top-level filter that applies simultaneously to all widgets. You can select one or more clusters, and one or more namespaces within selected clusters. When no clusters or namespaces are selected, the view automatically switches to *All*.  Any change to the filter is immediately reflected by all widgets, limiting the data they present to the selected scope. The *dashboard filter* does not affect the *status bar*.
+
+[id="widget-options"]
+==== Widget options
+
+Some widgets are now customizable to help you focus on what matters the most to you. Widgets offer different controls that change how the data is sorted, allow you to filter the data, and help you to customize the widget’s output. 
+
+Widgets offer two ways to customize different aspects:
+
+* An *Options* menu, when present,  provides specific options applicable to that widget. 
+* A *dynamic axis legend*, when present, allows you to filter data by hiding one or more of the axis categories. For example, in the *Policy violations by category* widget, you can click on a severity to include or exclude respective violations from the data.
+ 
+[NOTE]
+====
+Individual widget customization settings are short lived and are reset to the system default upon leaving the dashboard.
+====
+
+[id="new-widgets"]
+==== Actionable widgets
+
+The dashboard provides the following actionable widgets:
+
+* *Policy violations by severity*: This widget shows the distribution of violations across severity levels for the dashboard filtered scope. Clicking a *severity level* in the chart takes you to the *Violations* page, filtered for that severity and scope. It also lists the three most recent violations of a *Critical* level policy within the scope you defined in the *dashboard filter*. Clicking a specific violation takes you directly to the *Violations* detail page for that violation.
```

Comment:
@kcarmichael08 should "dashboard" have a capital D? And should "dashboard filtered scope" become "Dashboard filtered by scope"?